cache/models/samples.js

```diff
@@ -586,7 +596,7 @@ module.exports = {
    */
   bulkUpsertSample(sampleQueryBody) {
     const promises = sampleQueryBody.map(
-      (sampleReq) => upsertOneSample(sampleReq)
+      (sampleReq) => upsertOneSample(sampleReq, null, true)
     );
```

Comment:
While you are making these changes, can you add subjectAbsolutePath to the samples returned? I need it to attach the subject to the sample object.